tpl/partials/partials.go

```diff
@@ -72,6 +73,10 @@ func (ns *Namespace) Include(name string, contextList ...interface{}) (interface
 			b := bp.GetBuffer()
 			defer bp.PutBuffer(b)
 
+			if ns.deps.Cfg.GetBool("templateMetrics") {
+				defer ns.deps.Metrics.MeasureSince(templ.Name(), time.Now())
+			}
+
 			if err := templ.Execute(b, context); err != nil {
```

Comment:
I agree, but we're using `Execute` from external packages (like `html/template`).  Can you help build out the template handlers so that we can intercept the `Execute` calls?